### PR TITLE
Major updates to the codebase

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made

- Added Github Workflow __tested locally and in the forked repo__
- Added `.gitignore` file
- Moved all the scripts to the `/scripts` folder

## References

Issue [543 Update datasets](https://github.com/datopian/datahub-next/issues/543)

cc @anuveyatsu

### CODE REVIEW

File: data/language-codes-3b2.csv
Comment (anuveyatsu) [inline data/language-codes-3b2.csv:1]: @gradedSystem is it expected that this data file has 184 language codes removed?

File: data/language-codes-full.csv
Comment (anuveyatsu) [inline data/language-codes-full.csv:1]: @gradedSystem can you check here as well?

File: data/language-codes.csv
Comment (anuveyatsu) [inline data/language-codes.csv:1]: and here @gradedSystem

Comment (gradedSystem) [reply data/language-codes.csv:1]: FIXED

**Comment by gradedSystem** [comment]
cc @anuveyatsu Fixed the .sh file now it should be alright

**Pull request merged**